exercises/exercise_3/README.md

```diff
@@ -1,266 +1,255 @@
+    Usage:  docker app uninstall INSTALLATION_NAME [--target-context TARGET_CONTEXT] [OPTIONS]
+
+    Uninstall an application
+
+    Examples:
+    $ docker app uninstall myinstallation --target-context=mycontext
+
+    Options:
+          --credential-set stringArray   Use a YAML file containing a credential set or a credential set
+                                        present in the credential store
+          --force                        Force removal of installation
+          --target-context string        Context on which the application is installed (default: <current-context>)
+          --with-registry-auth           Sends registry auth
+    ```
+    </details>
+
+    We'll see that we need to specify the `INSTALLATION_NAME` and the `target-context`.
+
+2. Use the `docker app uninstall` command to remove the application.
+
+    <details>
+      <summary>Solution/Full Output</summary>
+    
+    ```console
+    $ docker app uninstall voting-app --target-context swarm
+    Removing service voting-app_db
+    Removing service voting-app_redis
+    Removing service voting-app_result
+    Removing service voting-app_vote
+    Removing service voting-app_worker
+    Removing network back-tier
+    Removing network front-tier
+    Application "voting-app" uninstalled on context "swarm"
+    ```
+    </details>
+
+    :tada: It's gone now!
+
+3. To verify, you can run `docker app ls --target-context swarm` and validate that it's gone.
+
+    <details>
+      <summary>Full output</summary>
+    
+    ```console
+    $ docker app ls --target-context swarm
+    INSTALLATION APPLICATION LAST ACTION RESULT CREATED MODIFIED REFERENCE
+    ```
+    </details>
+
+
+## Pushing our Docker App
+
+Now that we've learned how to deploy a locally-sourced application, let's push our app to Docker Hub and deploy from there!
+
+1. Let's first take a look at the various options and flags on the `docker app push` command.
+
+    <details>
+      <summary>Full output</summary>
+    
+    ```console
+    $ docker app push --help
+
+    Usage:  docker app push [APP_NAME] --tag TARGET_REFERENCE [OPTIONS]
+
+    Push an application package to a registry
+
+    Examples:
+    $ docker app push myapp --tag myrepo/myapp:mytag
+
+    Options:
+          --insecure-registries strings   Use HTTP instead of HTTPS when pulling from/pushing to those registries
+          --platform strings              For multi-arch service images, only push the specified platforms
+      -t, --tag string                    Target registry reference (default: <name>:<version> from metadata)
+    ```
+    </details>
+
+2. When pushing, we will tag the application to include your Docker Hub account (`<your-hub-username>/voting-app.dockerapp`) and specify the tag as the current version (which defaulted to `0.1.0`). While not required, we do encourage you to keep the `.dockerapp` suffix to the image name to help make it clear that that repo _is_ a Docker App bundle.
```

Comment:
totally nit: `make it clear that that repo` sounds strange 🤔 what about `make it clear that this repo` ?